etc/profile-a-l/device-flasher.linux.profile

```diff
@@ -0,0 +1,82 @@
+# Firejail profile for device-flasher.linux
+# Description: CalyxOS' device flasher
+# This file is overwritten after every install/update
+
+# Usage: run firejail ./device-flasher.linux in the folder with it and your factory image.
+
+# Warning!
+# ADB does not work under this sandbox, this is not a problem however, it just means you need to
+# reboot the device into bootloader mode first. To do this, hold Volume Down + Power until you get
+# into a screen with an Android robot being repaired. You'll need OEM Unlocking beforehand.
+
+# Fedora: you may need to manually install a third-party resource for udev rules, as Fedora ships broken ones on android-tools.
+
+quiet
+include device-flasher.linux.local
+# Persistent local customizations
+include globals.local
+# Persistent global definitions
+
+ignore noexec ${HOME}
+
+include allow-bin-sh.inc
+
+blacklist /opt
+blacklist /srv
```

Comment:
Added this only because of rusty-snake's kyst, works fine so might as well keep it?
I'm not entirely sure what /opt and /srv are for.

Reply:
> Added this only because of rusty-snake's kyst, works fine so might as well
> keep it?

Seems fine.

> I'm not entirely sure what /opt and /srv are for.

/opt is sometimes used for putting the entire installation files of third-party
programs into (as with `Program Files` in Windows).

/srv is used by things like HTTP servers.